src/main/java/array/ExerciseArray04.java

```diff
@@ -0,0 +1,36 @@
+package array;
+
+import java.util.Arrays;
+
+public class ExerciseArray04 {
+
+    public static int[] mergeTwoSortedArray(final int[] arr1, final int[] arr2) {
+
+        int i = arr1.length - 1;
+        int j = arr2.length - 1;
+        final int[] merge = new int[i + j + 2];
+        int k = merge.length - 1;
```

Comment:
Thử để vòng lặp xuất phát từ 0 thì logics sẽ thuận, dễ hiểu hơn


```suggestion
        final int[] merge = new int[arr1.length + arr2.length];
        
        
        int i = 0;
        int j = 0;
        int j = 0;
```